src/transformers/__init__.py

```diff
@@ -599,6 +600,24 @@
             "CamembertModel",
         ]
     )
+    _import_structure["models.character_bert"].extend(
+        [
+            "CHARACTER_BERT_PRETRAINED_MODEL_ARCHIVE_LIST",
+            "CharacterCnn",
+            "CharacterBertForMaskedLM",
+            "CharacterBertForMultipleChoice",
+            "CharacterBertForNextSentencePrediction",
+            "CharacterBertForPreTraining",
+            "CharacterBertForQuestionAnswering",
+            "CharacterBertForSequenceClassification",
+            "CharacterBertForTokenClassification",
+            "CharacterBertLayer",
+            "CharacterBertLMHeadModel",
+            "CharacterBertModel",
+            "CharacterBertPreTrainedModel",
+            "load_tf_weights_in_character_bert",
```

Comment:
I think those can be removed from the init :)

Reply:
Should I keep CharacterBertModel ?